File: program26.c
/*
0 -35 = fail
35 - 50 = pass class
50 - 60 = second class
60 - 70 = first class
70 - 100 = first class with distinction
*/



#include<stdio.h>

void DisplayClass(float fMarks)
{
    if((fMarks > 0) && (fMarks < 35.0))
    {
        printf("You are failed\n");
    }
    else if((fMarks >= 35) && (fMarks < 50.0))
    {
        printf("Pass class\n");
    }
    else if((fMarks >= 50.0) && (fMarks < 60.0))
    {
        printf("Second class\n");
    }
    else if((fMarks >= 60.0) && (fMarks < 70.0))
    {
        printf("first class\n");
    }
    else if((fMarks >= 70.0) && (fMarks <= 100.0))
    {
        printf("first class with distinction\n");
    }

}
int main()
{
    float fValue = 0.0;

    printf("Enter the percentage\n");
    scanf("%f",&fValue);

    DisplayClass(fValue);

    return 0;
}

File: program26.java
/*
0 -35 = fail
35 - 50 = pass class
50 - 60 = second class
60 - 70 = first class
70 - 100 = first class with distinction
*/

import java.util.Scanner;

public class program26
{
    
    public static void main(String arg[])
    {
        Scanner reader = new Scanner(System.in);

        System.out.print("Enter the percent:");
        int percent= reader.nextInt();

        if((percent > 0) && (percent < 35.0))
        {
            System.out.print("You are failed");
        }

        else if((percent > 35.0) && (percent < 50.0))
        {
            System.out.print("pass class");
        }

        else if((percent > 50.0) && (percent < 60.0))
        {
            System.out.print( "second class");
        }

        else if((percent > 60.0) && (percent < 70.0))
        {
            System.out.print("first class");
        }

        else if((percent > 70.0) && (percent < 100.0))
        {
            System.out.print("first class with distinction");
        }
        

        
    }
}